### ISSUE TITLE
Connect People With Community Grown Food

### ISSUE THREAD

**Issue by Farmer-Paige** [atxhack4change/2016-project-proposals#17]
## Short description of the idea (140 character tweet):

Quickly share or find a project where food is grown locally, affordably, and sustainably.

## What is your project concept or idea? What challenge or opportunity will it solve?

A lot of people are looking for ways to support food access initiatives or tell the community about their great project.

The Urban Patchwork website is meant to be a portal connecting community members with each other and opportunities to readily access food, projects, workshops, and more.

We have simplified our site, but we need to make it super easy for people to post a project page showing the awesome work they're doing and allowing people to find them. On the back-end, we need an easy to use dashboard that makes it easy for people to create and publish their project without a lot of support from our staff.

## Who will benefit from your project? Describe the humans at the center of the problem - who are they?

On the back end, the people who would use this are folks like the Festival Beach Food Forest, school gardens, neighborhood gardeners, foragers, and people providing community services that center around urban food production, availability, harvesting, and sharing.

On the user end, people who want to volunteer or donate, people who want to find where they can get food grown in their neighborhood, or people who want to start a neighborhood garden or orchard will find amazing projects to get connected to (and find sources of food growing in the city!)
 
### How would they benefit? Can you tell their story?

People who have great projects are sometimes unable to tell enough people about their projects. They would benefit from our support and the opportunity to reach more people. They would have an avenue for requesting volunteers and donations, and they could tell the community about the good work they're doing.

People who are looking for volunteering opportunities are often not sure where to go, who to contact, or can't find enough information about a project to know if it's right for them. We get calls and emails each week from people looking for internships, chances to help in their community, or to know how to start their own project.

People who are looking for alternative sources of food and opportunity in their neighborhood often don't know about opportunities that are right around the corner. With a beautifully published portal of opportunities that are focused on urban food production, people can find each other and find food.

### Do you have photos of them? Or quotes?

I will find some...much of that is buried in my head and on the server...which is why we need this portal on the website! :-)

## Issue area(s) relevant to your project idea: 

- Children & Youth
- Economic Opportunity
- Education
- Environment
- Food/Hunger
- Health
- Mobility/Transportation
- Neighborhoods
- Sustainability

## What is the current state of your project idea?

Somewhere between the following:

- I have an idea or **concept** of a solution, but have not yet prototyped or tested it
- I tested a **prototype** and want to build it

### Do you have any links to text, mockups, code, or data?

Yes, I have an existing website and project page template that I want the pages to fit into. A back end dashboard that looks and feels similar to Google forms would be a great example of what I want the user experience to be for people posting their projects

## Tell us about yourself. 

I'm the founder and director of Urban Patchwork - a food access and environmentally focused 501c3 that has existed since 2009. I believe strongly that food can and should be grown in the city where we live. When we are growing more food in the city our food system is more stable, our environment can get restored, and more people can have access to fresh, affordable food and economic opportunities for their families. I want to change how we think about food production in the city and in our culture today.

### What is your background? Why is this challenge important to you?

Food is a basic human right that many people do not have. Our current food system is almost completely outsourced, is extremely damaging to the environment, and food access is largely defined by the consumer based measurements for how much it costs for people to buy. This is all very unstable, unsustainable, disenfranchising, and disempowering. I aim to bring balance to our food system and connect people directly with food again.

### Are you representing a group in submitting this project idea? 

In a way, yes. I'm representing my organization as well as the community we serve.

**Comment by mateoclarke** [comment]
Thanks @Farmer-Paige! I've added some labels based on your responses. Didn't Urban Patchwork participate in the ATX Hack for Change a couple years ago on a similar project. Do you have links or resources related to that effort?

**Comment by madrusec** [comment]
This is an awesome idea! I have and idea a little bit along these lines as well (https://github.com/atxhack4change/2016-project-proposals/issues/22). Maybe we could come up with a unified idea that brings both together in some way (just a thought)?

**Comment by ssharif1** [comment]
### ACTION NEEDED 

Hello Project Champion, 

Please provide the following:
Full Name + Email Address 
You can comment here or send to ssharif1@stedwards.edu 

We are going to be sending out a note + newsletter to our 345 (as of May 18) attendees sharing our submitted projects before the event and your hackers will want to know who you are. Additionally, we have PC-specific communication that we want to make sure that you are included on.

Thanks,
ATX H4C Staff

**Comment by ssharif1** [comment]
Hello @Farmer-Paige, 

Please provide the following so we can finalize your project in our review process:
Full Name + Email Address 
You can comment here or send to ssharif1@stedwards.edu

Thanks,
ATX H4C Staff

**Comment by Farmer-Paige** [comment]
Hi Mateo,

I'm so sorry I just saw your earlier email. I've been out of town and neck
deep in projects.

A couple years ago we worked with the Hackathon to build an interactive
mapping program for fruit and nut trees. That effort is now "owned" and
managed by TreeFolks.

This proposed project is for community members to use our website to post
their current and proposed projects and engage with each other among other
things.

However, I think it will be best if I withdraw my project at this time. I
am now needing to go out of town the weekend of the event and I'm the best
champion from our organization. Since I can't be there we should wait.

Thank you so much for reaching out, and I hope you have a great hackathon!!

-Paige


------------   Paige Hill Oliverio   ------------
urbanpatchwork.com <http://www.urbanpatchwork.com/> | urbanpatchwork.org
*Growing Food and Community in the City*
512-662-1854
